components/remote/ui/ir/CardButtonsTransport.qml

```diff
@@ -29,91 +31,118 @@ Item {
 
-        Button {
-            visible: obj.isSupported(Remote.F_PREVIOUS)
-            title: qsTr("Prev") + translateHandler.emptyString
-            mouseArea.onClicked: { obj.previous(); }
+        Row {
+            Layout.alignment: Qt.AlignHCenter
+            spacing: 32
+            IconButton {
+                visible: obj.isSupported(Remote.F_BACKWARD)
+                image.source: "qrc:/images/remote/fast_rewind-white-48dp.svg"
+                mouseArea.onClicked: { obj.backward(); }
+            }
+
+            IconButton {
+                visible: obj.isSupported(Remote.F_PAUSE)
+                image.visible: true
+                image.source: "qrc:/images/remote/pause-white-48dp.svg"
+                mouseArea.onClicked: { obj.pause(); }
+            }
+
+            IconButton {
+                visible: obj.isSupported(Remote.F_FORWARD)
+                image.source: "qrc:/images/remote/fast_forward-white-48dp.svg"
+                mouseArea.onClicked: { obj.forward(); }
+            }
         }
 
-        Button {
-            visible: obj.isSupported(Remote.F_NEXT)
-            title: qsTr("Next") + translateHandler.emptyString
-            mouseArea.onClicked: { obj.next(); }
-        }
-
-        Button {
-            visible: obj.isSupported(Remote.F_INFO)
-            title: qsTr("Info") + translateHandler.emptyString
-            mouseArea.onClicked: { obj.info(); }
-        }
-
-        Button {
-            visible: obj.isSupported(Remote.F_BACKWARD)
-            title: qsTr("BWD") + translateHandler.emptyString
-            mouseArea.onClicked: { obj.backward(); }
-        }
-
-        Button {
-            visible: obj.isSupported(Remote.F_FORWARD)
-            title: qsTr("FWD") + translateHandler.emptyString
-            mouseArea.onClicked: { obj.forward(); }
-        }
-
-        Button {
-            visible: obj.isSupported(Remote.F_RECORD)
-            title: qsTr("Rec") + translateHandler.emptyString
-            mouseArea.onClicked: { obj.record(); }
-        }
-
-        Item { width: 118; height: 70 }
-
-        Button {
-            visible: obj.isSupported(Remote.F_EXIT)
-            title: qsTr("Exit") + translateHandler.emptyString
-            mouseArea.onClicked: { obj.exit(); }
-        }
-
-        Item { width: 118; height: 70 }
-
-        Button {
-            visible: obj.isSupported(Remote.F_BACK)
-            title: qsTr("Back") + translateHandler.emptyString
-            mouseArea.onClicked: { obj.back(); }
+        Row {
+            Layout.alignment: Qt.AlignHCenter
+            spacing: 32
+            IconButton {
+                visible: obj.isSupported(Remote.F_PREVIOUS)
+                image.source: "qrc:/images/remote/skip_previous-white-48dp.svg"
+                mouseArea.onClicked: { obj.previous(); }
+            }
+
+            IconButton {
+                visible: obj.isSupported(Remote.F_INFO)
+                image.source: "qrc:/images/remote/info-white-48dp.svg"
```

Comment:
For icons, best would be to extend the icons.tft font file with new icons and reference them from Style.qml.